source/fuzz/fuzzer_pass.h

```diff
@@ -179,7 +179,7 @@ class FuzzerPass {
   // on the |is_irrelevant| parameter.
   uint32_t FindOrCreateIntegerConstant(const std::vector<uint32_t>& words,
                                        uint32_t width, bool is_signed,
-                                       bool is_irrelevant);
+                                       bool is_irrelevant = false);
```

Comment:
Looks like you may have merged incorrectly here; @Vasniktel changed this so that it does not use default parameters anymore, so we should keep it that way.

Reply:
I have now removed the default parameter (I also noticed that the last line in the comment above seemed to break pointlessly, so I fixed it)